packages/fx-core/src/plugins/resource/sql/constants.ts

```diff
@@ -108,4 +109,7 @@ export class AzureSqlBicep {
 export class AzureSqlBicepFile {
   static readonly moduleTemplateFileName: string = "provision.template.bicep";
   static readonly ProvisionModuleTemplateFileName = "sqlProvision.template.bicep";
+  static readonly newDatabaseModuleTemplateFileName: string =
+    "newDatabase.provision.template.bicep";
+  static readonly newDatabaseProvisionModuleTemplateFileName = "newDatabase.template.bicep";
```

Comment:
The two values and names don't seem to match

Reply:
Sure. Updated it. Thanks.